src/wallet/rpcwallet.cpp

```diff
@@ -2714,6 +2714,16 @@ UniValue mint(const UniValue& params, bool fHelp)
             "\nAutomatically choose denominations to mint by amount\n" +
             HelpRequiringPassphrase());
 
+    // Ensure Sigma mints is already accepted by network so users will not lost their coins
+    // due to other nodes will treat it as garbage data.
+    {
+        LOCK(cs_main);
```

Comment:
Is it necessary to lock cs_main here?

Reply:
Yes because we want to access `chainActive`.